x-pack/plugins/synthetics/public/legacy_uptime/state/actions/monitor_management.ts

```diff
@@ -18,6 +18,9 @@ import { SyntheticsServiceAllowed } from '../../../../common/types';
 export const getMonitors = createAction<FetchMonitorManagementListQueryArgs>(
   'GET_MONITOR_MANAGEMENT_LIST'
 );
+
+export const deletedMonitorAction = createAsyncAction<string, string>('DELETE_MONITOR');
+
```

Comment:
Redundant action. The one used is defined in `.../synthetics/public/legacy_uptime/state/actions/delete_monitor.ts`